src/lib/application-state-stores/explorer-stores.ts

```diff
@@ -0,0 +1,125 @@
+import { Readable, writable } from "svelte/store";
+import type { MetricsExplorerEntity } from "$lib/redux-store/explore/explore-slice";
+import type { MetricViewMetaResponse } from "$common/rill-developer-service/MetricViewActions";
+import type { TimeSeriesTimeRange } from "$common/database-service/DatabaseTimeSeriesActions";
+
+export interface MetricsExplorerStoreType {
+  entities: Record<string, MetricsExplorerEntity>;
+}
+const { update, subscribe } = writable({
+  entities: {},
+} as MetricsExplorerStoreType);
+
+const UpdateMetricsExplorer = (
```

Comment:
Nit: what's with the capitalization for `UpdateMetricsExplorer`, `Methods`, and `MetricsExplorerStore`? Is that a convention for stores? We don't do it elsewhere AFAIK